alacritty_config_derive/src/config_deserialize/de_enum.rs

```diff
@@ -14,7 +14,16 @@ pub fn derive_deserialize(ident: Ident, generics: Generics, data_enum: DataEnum)
     for variant in data_enum.variants.iter().filter(|variant| {
         // Skip deserialization for `#[config(skip)]` fields.
         variant.attrs.iter().all(|attr| {
-            !crate::path_ends_with(&attr.path, "config") || attr.tokens.to_string() != "(skip)"
```

Comment:
Can this function be removed?

Reply:
I think I've tried, but it's actually required in other place. I could probably inline it there, but it looks sort of ugly.